apps/right-to-rent-check/index.js

```diff
@@ -36,28 +39,34 @@ module.exports = {
       fields: ['living-status'],
       next: '/tenancy-start',
       forks: [{
-        target: '/person-location',
+        target: '/tenant-in-UK',
```

Comment:
please don't use mixed-case in the url's. https://www.w3.org/TR/WD-html40-970708/htmlweb.html